tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/otextension/CoteShared.java

```diff
@@ -0,0 +1,43 @@
+package dk.alexandra.fresco.tools.ot.otextension;
+
+import dk.alexandra.fresco.framework.util.ByteArrayHelper;
+import dk.alexandra.fresco.framework.util.Drbg;
+import dk.alexandra.fresco.framework.util.PaddingAesCtrDrbg;
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Superclass containing the common methods for the sender and
+ * receiver parties of correlated OT with errors.
+ */
+public class CoteShared {
```

Comment:
I am not a huge fan of these `*Shared` super classes. It seems to me they are mainly used to hold common parameters and utility methods which could just as well be held in a separate helper class (basically the resourcepool). 

At least, I think they should be made `abstract` as only their subclasses are intended to be instantiated. 
(This comment goes for essentially all the classes with the `Shared` suffix).

Reply:
I have already removed the getter methods so that they now only contain common methods in order to avoid code-copy between the sender and receiver, while still keeping an object oriented approach.
I have now also made them abstract in commit faa7e12f